api/bag_dataset.cpp

```diff
@@ -1055,6 +1056,11 @@ std::tuple<double, double> Dataset::gridToGeo(
     return {x, y};
 }
 
+void handleAbrt(int signum) {
```

Comment:
I think this runs into some problems that @selimnairb and I talked about during the last meeting, I'm working on some code now to handle it "safely" within the specification. The worry is that using the std::cerr and streams allocates, which isn't well defined behavior within an abort handler. I wonder if there's a way that we can tell aborts apart from segfaults?